bin/compile

```diff
@@ -77,13 +77,20 @@ echo -n "-----> Installing ivysettings.xml....."
 if [ -f .ivy2/ivysettings.xml ]; then
   rm .ivy2/ivysettings.xml
 fi
+
 mkdir -p .ivy2
-PLAY_SETTINGS_URL="http://s3.amazonaws.com/heroku-jvm-langpack-play/ivysettings.xml"
-CREDS=$(echo '  <credentials host="maven.lendup.com" realm="closed site" username="${LENDUP_MAVEN_REPO_USER}" passwd="${LENDUP_MAVEN_REPO_PW}"/>' \
-  | perl -p -e 's/\$\{([^}]+)\}/defined $ENV{$1} ? $ENV{$1} : $&/eg; s/\$\{([^}]+)\}//eg')
-# NOTE: If testing on mac: brew install gnu-sed; alias sed=gsed
-curl --silent --max-time 10 --location $PLAY_SETTINGS_URL | sed "/<ivysettings>/ a ${CREDS}" > .ivy2/ivysettings.xml
-echo " done"
+
+if [[ -e "conf/ivysettings.xml" ]] ; then
+  # If the app has its own ivysettings.xml, use that.
+  cp conf/ivysettings.xml .ivy2/.
```

Comment:
What are you going to do about the maven repo creds here?

Reply:
they'll come from the environment during slug build time (see the ivysettings.xml in the other PR). not sure yet how that's going to play with (e.g.) review apps but i believe there is a way to have it reference runtime config vars during build. still investigating that part